test/service.cc

```diff
@@ -4,180 +4,11 @@
 #include "service.h"
 #include "unifier.h"
 #include "genotyper.h"
+#include "utils.cc"
```

Comment:
It's unusual to `#include` a `.cc` source file like this; this may lead to duplication of the compiled code in the final executable. In this case it's fine to keep it this way as a matter of convenience since the executable is just the unit test program, but I just wanted to point it out.